arches_lingo/src/arches_lingo/components/scheme/SchemeHeader/components/ExportThesauri.vue

```diff
@@ -0,0 +1,243 @@
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import { useToast } from "primevue/usetoast";
+import Button from "primevue/button";
+import Dialog from "primevue/dialog";
+import InputText from "primevue/inputtext";
+import ProgressSpinner from "primevue/progressspinner";
+import RadioButton from "primevue/radiobutton";
+import SelectButton from "primevue/selectbutton";
+
+import { exportThesaurus } from "@/arches_lingo/api.ts";
+import {
+    DEFAULT_TOAST_LIFE,
+    DEFAULT_ERROR_TOAST_LIFE,
+    ERROR,
+} from "@/arches_lingo/constants.ts";
+
+const { $gettext } = useGettext();
+const toast = useToast();
+
+const props = defineProps<{
+    resourceId: string;
+    resourceName: string | undefined;
+}>();
+
+const loading = ref(false);
+const visible = ref(true);
+
+const exportDepth = ref("complete");
```

Comment:
This probably doesn't need to exist as the depth is derived from the root resource you choose and it was expressed that we did not envision the use case for exporting a single concept outside of the context of a hierarchy. Can you confirm though?